packages/lib/models/Setting.ts

```diff
@@ -1010,6 +1010,25 @@ class Setting extends BaseModel {
 				isGlobal: true,
 			},
 
+			// Enables/disables spellcheck in the mobile markdown beta editor.
+			'editor.mobile.spellcheckEnabled': {
+				value: true,
+				type: SettingItemType.Bool,
+				section: 'note',
+				public: true,
+				appTypes: [AppType.Mobile],
+				show: (settings: any) => settings['editor.beta'],
+				label: () => 'Enable spellcheck in the beta editor',
+				description: () => {
+					if (shim.mobilePlatform() === 'ios') {
+						return 'Enables automatic spellcheck in the beta editor. This may not work properly on iOS';
```

Comment:
Hmm, I don't think we should bother with this, it's just more strings to translate. Let's see what feedback we have.